packages/react-components/src/components/UnsupportedEnvironment.tsx

```diff
@@ -25,6 +28,8 @@ export interface UnsupportedEnvironmentStrings {
   secondaryText: string;
   /** String to display in the text for the help link */
   moreHelpLinkText: string;
+  /** String for continue without updating button */
```

Comment:
nit: be consistent in naming, you say `continue without updating button` here but the prop refers to `continueAnywayButton`
```suggestion
  /** String for continue anyway button */
```